client/gutenberg/extensions/mailchimp/editor.scss

```diff
@@ -18,4 +18,12 @@
 		display: none;
 	}
 
+	.wp-block-jetpack-mailchimp_text-input, .jetpack-submit-button {
+		margin-bottom: 1.5em;
```

Comment:
Let's use `rem` instead. I didn't realise the button was using a different font-size.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/31142/commits/c9e2061721f42c908e64d62e6b443469b542e0e5

Reply:
Perfect 💯